namesilo.php

```diff
@@ -2196,6 +2196,7 @@ public function checkAvailability($domain)
         }
 
         $response = $result->response();
+        $this->processResponse($api, $result);
```

Comment:
I'd like to see this moved above the condition on line 2194.  That way we'll still log the request even when to fails